### PR TITLE
[Redesign]Minor Fixes from functional pass

### PR DESCRIPTION
**Pull request by ryuyu**
Fixes the following issues:
 * Package Statistics & Package Version Statistics
   * Title gets cut off by the graph
   * The underline on hover for "Show more" covers a couple characters in front of "Show"
 * Statistics Page
   * Replace the subtitle with the bottom text of "Statistics last updated..." to make it consistent with the other statistics pages.
 * Upload
   * Error messages does not seem to be appearing for invalid package uploads.
   * Progress bar looks ~1px too short of being flush with the browse button
   * Empty Dependencies should probably still display something.

@joelverhagen @jonwchu @loic-sharma @scottbommarito

### CODE REVIEW

File: src/Bootstrap/less/theme/page-statistics-per-package.less [hunk 1]
```diff
@@ -1,5 +1,11 @@
 .page-stats-per-package
 {
+    .btn-expander {
+        &:hover, &:focus {
+        text-decoration: none;
```
Comment (loic-sharma) [inline src/Bootstrap/less/theme/page-statistics-per-package.less]: Missing a tab here.

Comment (ryuyu) [inline src/Bootstrap/less/theme/page-statistics-per-package.less]: Fixed.

File: src/Bootstrap/less/theme/page-statistics-per-package.less [hunk 2]
```diff
@@ -1,5 +1,11 @@
 .page-stats-per-package
 {
+    .btn-expander {
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/page-statistics-per-package.less]: This logically conflicts with another comment from the PM pass:

> Hover behavior over "Show more/less" is not consistent with other pages changes colors here, other pages, underlines (see Package Statistics). Should be consistent with other links (underline)

Is there another way to address this?

File: src/NuGetGallery/Controllers/AppController.cs [hunk 3]
```diff
@@ -48,6 +48,17 @@ protected internal virtual ActionResult SafeRedirect(string returnUrl)
             return new SafeRedirectResult(returnUrl, Url.Home());
         }
 
+        protected internal JsonResult Json(int statusCode, object obj)
```
Comment (joelverhagen) [inline src/NuGetGallery/Controllers/AppController.cs]: Leave an XML doc that the real purpose of this method is to set `TrySkipIisCustomErrors`. The `statusCode` is an unnecessary convenience.

File: src/NuGetGallery/Controllers/PackagesController.cs [hunk 4]
```diff
@@ -213,29 +213,27 @@ public virtual JsonResult UploadPackageProgress()
         [RequiresAccountConfirmation("upload a package")]
         public virtual async Task<JsonResult> UploadPackage(HttpPostedFileBase uploadFile)
         {
+            Response.TrySkipIisCustomErrors = true;
```
Comment (joelverhagen) [inline src/NuGetGallery/Controllers/PackagesController.cs]: We don't need this right?

File: src/NuGetGallery/Controllers/PackagesController.cs [hunk 5]
```diff
@@ -1086,12 +1070,11 @@ public virtual ActionResult Edit(string id, string version)
...
                 {
                     ModelState.AddModelError("Edit.VersionTitle", ex.Message);
-                    Response.StatusCode = 400;
-                    return Json(new string[] { ex.Message });
+                    return Json(400, new string[] { ex.Message });
                 }
             }
 
-            return Json(new
+            return Json(200, new
```
Comment (joelverhagen) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Can we just use the normal `Json` method here?

File: src/NuGetGallery/Controllers/PackagesController.cs [hunk 6]
```diff
@@ -394,15 +384,14 @@ public virtual JsonResult UploadPackageProgress()
...
-                    return Json(new string [] { ex.GetUserSafeMessage() });
+                    
+                    return Json(400, new string [] { ex.GetUserSafeMessage() });
                 }
             }
 
             var model = new VerifyPackageRequest(packageMetadata);
 
-            return Json(model);
+            return Json(200, model);
```
Comment (joelverhagen) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Original `Json(model)` method here?

File: src/NuGetGallery/Controllers/PackagesController.cs (hunk 5 above)
Comment (scottbommarito) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Yeah, I think `200` is implied in the base `Json(...)` method.

Comment (ryuyu) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Done.

File: src/NuGetGallery/Controllers/PackagesController.cs (hunk 6 above)
Comment (ryuyu) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Done

File: src/NuGetGallery/Controllers/PackagesController.cs (hunk 4 above)
Comment (ryuyu) [inline src/NuGetGallery/Controllers/PackagesController.cs]: correct. Removed.

**Pull request merged**